### DISCUSSION TITLE
Difference between h2ogpt-oig-oasst1-512-6.9b and h2ogpt-oasst1-512-12b

### DISCUSSION THREAD

**Discussion by karthikgali** [h2oai/h2o-llmstudio#93] [Q&A] [answered]
Hi,

What is the difference between h2ogpt-oig-oasst1-512-6.9b and h2ogpt-oasst1-512-12b. I am planning to use it as an alternative for chatgpt.

Regards,
Karthik

**Comment by pascal-pfeiffer** [comment, marked as answer]
These checkpoints follow the naming convention of h2oGPT-<DATASET_USED>-<TRAINING_TOKEN_LENGTH>-<PARAMETER_COUNT>
Both have beed trained with the [h2oGPT](https://github.com/h2oai/h2ogpt) repository.
